src/pytac/__init__.py

```diff
@@ -26,7 +31,12 @@
 """Error 402 is suppressed as we cannot import these modules at the top of the
```

Comment:
Looks like you have changed the directive this is referring to

Reply:
I'm assuming it is designed to be below the `from . import` statement? Equally, the comment should be updated from E402 to isort:skip?